odoo/addons/base/models/assetsbundle.py

```diff
@@ -197,39 +210,33 @@ def to_node(self, css=True, js=True, debug=False, async_load=False, defer_load=F
                 ["type", "text/javascript"],
                 ["data-src" if lazy_load else "src", src],
                 ['data-asset-bundle', self.name],
-                ['data-asset-version', self.version],
+                ['data-asset-version', self.version('js')],
                 ['onerror', '__odooAssetError=1']
             ])
             response.append(("script", attr, None))
 
         return response
 
-    @func.lazy_property
-    def last_modified_combined(self):
-        """Returns last modified date of linked files"""
-        # WebAsset are recreate here when a better solution would be to use self.stylesheets and self.javascripts
-        # We currently have no garanty that they are present since it will depends on js and css parameters
-        # last_modified is actually only usefull for the checksum and checksum should be extension specific since
-        # they are differents bundles. This will be a future work.
-
-        # changing the logic from max date to combined date to fix bundle invalidation issues.
-        assets = [WebAsset(self, url=f['url'], filename=f['filename'], inline=f['content'])
-            for f in self.files
-            if f['atype'] in ['text/sass', "text/scss", "text/less", "text/css", "text/javascript", "text/xml"]]
-        return ','.join(str(asset.last_modified) for asset in assets)
-
-    @func.lazy_property
-    def version(self):
-        return self.checksum[0:7]
+    def version(self, asset_type):
```

Comment:
The name `version` is perfect for a property, but not for a method.  What about `get_version` instead?  This would make the conversion less confusing, wouldn't it?